<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ArcGate2168" failures="1" tests="13" name="sila_test_cases.LinkAccount" time="10.756" errors="0" timestamp="2020-10-07T17:17:48 IST" skipped="0">
  <testcase classname="sila_test_cases.LinkAccount" name="Test_01_Link_account_with_all_valid_data" time="1.590"/>
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_02_Link_account_with_empty_user_handle" time="0.462"/>
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_03_Link_account_with_unregistered_user_handle" time="0.552"/>
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_04_Link_account_with_empty_Public_Token" time="0.562"/>
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_05_Link_account_with_empty_account_name" time="0.616">
    <failure message="expected [200] but found [400]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at sila_test_cases.LinkAccount.Test_05_Link_account_with_empty_account_name(LinkAccount.java:86)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Test_05_Link_account_with_empty_account_name -->
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_06_Link_account_with_1_char_account_name" time="1.357"/>
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_07_Link_account_with_41_chars_account_name" time="0.543"/>
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_08_Link_account_with_40_chars_account_name" time="1.594"/>
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_09_Link_account_with_invalid_signature" time="0.631"/>
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_10_Link_account_with_invalid_Public_Token" time="0.901"/>
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_11_Link_account_with_direct_link_account_success" time="0.691"/>
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_12_Link_direct_link_account_with_empty_user_handle" time="0.625"/>
  <system-out/>
  <testcase classname="sila_test_cases.LinkAccount" name="Test_13_Link_direct_link_account_with_unregistered_user_handle" time="0.632"/>
  <system-out/>
</testsuite> <!-- sila_test_cases.LinkAccount -->
